tests/testTriggers.go

```diff
@@ -731,6 +811,61 @@ func TriggerRebootNodes(contexts *[]*scheduler.Context, recordChan *chan *EventR
 	})
 }
 
+func randIntn(n, maxNo int) []int {
+	if n > maxNo {
+		n = maxNo
+	}
+	rand.Seed(time.Now().UnixNano())
+	generated := make(map[int]bool)
+	for i := 0; i < n; i++ {
+		randUniq := rand.Intn(maxNo)
+		if !generated[randUniq] {
+			generated[randUniq] = true
+		}
+	}
+	generatedNs := make([]int, 0)
+	for key := range generated {
+		generatedNs = append(generatedNs, key)
+	}
+	return generatedNs
+}
+
+func getNodesByChaosLevel(triggerType string) []node.Node {
```

Comment:
@thiguetta you are mixing up number nodes to restart with interval . How do we make both configurable ? I want to keep the frequency same but would like to keep rebooting different number of nodes. Can you please come up with that scheme with another parameter in configmap ? Should be picking up storage / storage less nodes equally.